README.md

```diff
@@ -1,20 +1,30 @@
-# torchao: PyTorch Architecture Optimization
-
 **Note: This repository is currently under heavy development - if you have suggestions on the API or use-cases you'd like to be covered, please open an github issue**
 
-The `torchao` package allows you to quantize and prune your models using native PyTorch.
+## Introduction
 
-The repo hosts both
-1. lower precision [dtypes](./torchao/dtypes) such as nf4, uint4
-2. Quantization [algorithms](./torchao/quantization) such as dynamic quant, smoothquant
-3. Sparsity [algorithms](./torchao/sparsity) such as Wanda
+torchao is a PyTorch native library for optimizing your models using lower precision dtypes, techniques like quantization and sparsity and performant kernels.
 
-## Success stories
-Our kernels have has been used to achieve SOTA inference performance on
+The library provides
+1. Support for lower precision [dtypes](./torchao/dtypes) such as nf4, uint4 that are torch.compile friendly
+2. Quantization [algorithms](./torchao/quantization) such as dynamic quant, smoothquant, GPTQ that run on CPU/GPU and Mobile.
+3. Sparsity [algorithms](./torchao/sparsity) such as Wanda that help improve accuracy of sparse networks
+4. Integration with other PyTorch native libraries like TorchTune and ExecuTorch
 
-1. Image segmentation modelss with [sam-fast](pytorch.org/blog/accelerating-generative-ai)
-2. Language models with [gpt-fast](pytorch.org/blog/accelerating-generative-ai-2)
-3. Diffusion models with [sd-fast](pytorch.org/blog/accelerating-generative-ai-3)
+## Key Features
+* Native PyTorch techniques, composable with torch.compile
+* APIs tested on following hardware - A100, T4(colab) 
+* Supports the following eager quantization techniques. 
```

Comment:
so which techniques are compile specific and which are eager? If it's the same set then we can set say something like kernels that work with both eager and compile such as: int8 etc..

Reply:
I think they all work with both. @HDCharles or @jerryzh168 can you confirm?